test/integration/webhook/v1alpha2/clusterqueue_test.go

```diff
@@ -148,6 +165,22 @@ var _ = ginkgo.Describe("ClusterQueue Webhook", func() {
 			gomega.Expect(errors.IsForbidden(err)).Should(gomega.BeTrue(), "error: %v", err)
 		})
 
+		ginkgo.It("Should have valid flavor quantity when creating", func() {
+			ginkgo.By("Creating a new clusterQueue")
+			cq := func() *kueue.ClusterQueue {
+				res := testing.MakeResource("cpu")
+				for i := 0; i < 17; i++ {
```

Comment:
Maybe you put the numbers like `16` `8` in test/integration/framework/constants.go, change 17 to `FlavorMaxItems` + 1?
It will help to improve the readability of code.